app/domain/authentication/authn_jwt/configuration_jwt_generic_vendor.rb

```diff
@@ -2,7 +2,9 @@ module Authentication
   module AuthnJwt
     # Mock JWTConfiguration class to use it to develop other part in the jwt authenticator
     class ConfigurationJWTGenericVendor < ConfigurationInterface
+
       def initialize
+        @token_fetcher = Authentication::AuthnJwt::FetchTokenFromCredentials.new
```

Comment:
i think FetchTokenFromCredentials should be a command class, @orenbm, @abrahamko do you agree ?

Reply:
Yes, this looks like a pure action, so command class fits perfectly.
Unless there's a good reason why not; is there, @sashaCher?
Also, it amplifies the dependencies, as they are explicitly declared at the top of the class, so no hidden dependency such as `Authentication::Jwt::DecodedCredentials`.

Reply:
I've converted it to a command class.
Thank you

Reply:
can you also call the parameter `fetch_token_from_credentials`? the param name shoulod match the class it represents